index.md

```diff
@@ -1,4 +1,4 @@
-_“Best practice templates and writing instructions for documenting open source software.”_
+_“Bests for documenting software.”_
```

Comment:
```suggestion
_“Best practices for documenting software.”_
```

But I believe there is further discussion on the ML, and we should wait until we have agreement.